lib/KalturaModel.php

```diff
@@ -66,6 +66,21 @@ public function getClientSideSession( $privileges = '', $expiry = 86400 ) {
 		return sanitize_text_field( $this->createKS( $this->_partnerId, $this->_userId, Kaltura_Client_Enum_SessionType::USER, $privileges, $expiry ) );
 	}
 
+	public function getAdminSessionUsingApi($partnerId, $adminSecret, $privileges = "", $expiry = 86400)
```

Comment:
I changed the signature because I wanted that the partner ID and adminSecret to be calculated once (in the `infoAction()`).
Also, these methods originally used `get_option`, which, as I explained in [this comment](https://github.com/kaltura/all-in-one-video-pack.wordpress/pull/46#discussion_r77508628), is not what we want all the time.